src/components/LineChart/index.tsx

```diff
@@ -66,7 +71,7 @@ export const LineChart = withTooltip<LineGraphType, DateValueType>(
       scale: xScale,
       values: data.map(el => el.date),
       // TODO: review how to switch to a 24 hour display
-      tickFormat: (v: Date) => timeFormat("%H:%M")(v),
+      //tickFormat: (v: Date) => timeFormat("%H:%M")(v),
```

Comment:
not sure if this was meant to be commented out, but might aswell remove if unused

Reply:
Absolutely. It was unused because the `tickFormat` needs to be applied to the `AxisBottom` in this case, so it didn't do anything anyway.